src/Components/Search/style.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css?family=IBM+Plex+Sans&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Martel+Sans&display=swap");
```

Comment:
I'd setup fonts once for the whole app, so you won't need to import it every time